web-components/graphrag-widget/README.md

```
@@ -0,0 +1,108 @@
+![Preact](https://img.shields.io/badge/Preact-10.x-673AB8)
+![npm](https://img.shields.io/npm/v/@falkordb/graphrag-widget)
+
+<p align="center">
+  <img src="docs/demo.gif" alt="FalkorDB GraphRAG Widget Demo" width="640" />
+</p>
+
+---
+
+## How It Works
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)